install.sh

```diff
@@ -171,7 +176,7 @@ function install() {
         done
     done
     if [ "$SHOULD_POST_INSTALL" = true ]; then
-        ./post-install.sh --dest-dir "$DEST_DIR" --sysconf-dir "$SYSCONF_DIR"
+      ./post-install.sh --dest-dir "$DEST_DIR" --sysconf-dir "$SYSCONF_DIR" $( (( NO_SUDO == true )) && echo "--no-sudo" )
```

Comment:
for safety, should be replaced by `"$([ "$NO_SUDO" == true ] && echo "--no-sudo")"`